src/Core/src/Core/IPage.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.Maui
+{   /// <summary>
+	/// Represents a .NET MAUI Page.
+	/// </summary>
+	public interface IPage : IContentView
```

Comment:
I don't think we want to create an `IPage` for this. Can we follow the same route as we do for things like `ITitledElement`?  I feel like the interface for the background image needs to be more interesting then just having the `IImageSource` as well. Perhaps we use something like

`interface IViewBackgroundImagePart : IImageSourcePart` because then that makes it more compatible with other ImageLoading structures

You can then just instantiate a

```C#
		public ImageSourcePartLoader SourceLoader =>
			_imageSourcePartLoader ??= new ImageSourcePartLoader(this, () => VirtualView, OnSetImageSource);
```

And use that to handle all the loading behavior. 

At that point the mapper could possibly even just be on ViewHandler and thus a generalized BackgroundImage for native views is born

Reply:
Removed `IPage` and implemented `IViewBackgroundImagePart`. Also added a ContentPage Gallery sample where easily test this scenario.

Reply:
@PureWeen Could you review the PR again?. Thnx!